packages/marqus-desktop/src/renderer/components/Monaco.tsx

```diff
@@ -289,6 +289,16 @@ const StyledEditor = styled.div`
   height: calc(100% - ${TOOLBAR_HEIGHT});
 `;
 
+export function createMarkdownModel(
+  content: string | undefined,
+): monaco.editor.ITextModel {
+  return monaco.editor.createModel(
+    content ?? "",
+    // N.B. Language needs to be specified for syntax highlighting.
+    "markdown",
+  );
+}
+
```

Comment:
This bug took me awhile to crack so I pulled the logic out to a separate function to make it easier to test (really didn't want this to potentially regress).